Patches/ExilePatch.cs

```diff
@@ -170,33 +170,27 @@ static void WrapUpPostfix(GameData.PlayerInfo exiled)
         if (Witch.IsEnable) 
             Witch.RemoveSpelledPlayer();
 
-        if (HexMaster.IsEnable)
-            HexMaster.RemoveHexedPlayer();
-
```

Comment:
Seems `HexMaster.RemoveHexedPlayer();` was removed

Reply:
I thought you removed it in your pr...

Reply:
No

![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/18b43d86-6498-4262-933b-2baad1c56b3b)

Reply:
Pls revert it
I dont have permission to directly write into lezaiya 's fork